db/migrate/20200731213827_add_acuant_result_to_proofing_costs.rb

```diff
@@ -0,0 +1,5 @@
+class AddAcuantResultToProofingCosts < ActiveRecord::Migration[5.2]
+  def change
+    add_column :proofing_costs, :acuant_result_count, :integer, default: 0
```

Comment:
I don't see the new column referenced in the code, does it get inferred or automatically updated somewhere?

Reply:
It's dynamically referenced in `add_user_proofing_cost.rb` --- I'm adding a spec shortly that checks the results so hopefully it'll be a little clearer